pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
 
 overrides:
```

Comment:
The changes to pnpm-lock.yaml really should not be in this PR. They probably are the reason that I am having issues with the shell